src/converter/internal/convertercontroller.cpp

```diff
@@ -117,6 +119,13 @@ Ret ConverterController::fileConvert(const muse::io::path_t& in, const muse::io:
         return make_ret(Err::InFileFailedLoad);
     }
 
+    if (optionsJson.length() != 0) {
+        ret = BackendApi::applyTranspose(notationProject->masterNotation()->notation(), optionsJson);
```

Comment:
Introducing a dependency from `ConverterController` on `BackendApi` is not ideal (albeit better than duplicating code, as you already said). It may be better to move `applyTranspose` to a new utility class, say `src/converter/internal/converterutils.{h,cpp}`, and then reuse it in both `BackendApi` and `ConverterController`.

Reply:
Done